draft-ietf-quic-load-balancers.md

```diff
@@ -324,6 +308,78 @@ the packet. The number of servers is usually consistent on the time scale of a
 QUIC connection handshake. Another might simply hash the address/port 4-tuple.
 See also {{version-invariance}}.
 
+Note that any information used by the Fallback Algorithm might not be repeated
+from one QUIC packet to another. The 4-tuple is assumed to remain stable during
+the handshake, but the server can change the destination connection ID after the
+client's first flight.
+
```

Comment:
Sure. But if the server changes the CID, isn't it expected to pick a routable CID?

Reply:
If it doesn't have a config, it uses 0b111, which is now defined as unroutable. That's the big change in this PR.

Reply:
OK.